### PR TITLE
feat: :sparkles: grid export for loop

### PR DESCRIPTION
**Pull request by espenkalle**

### CODE REVIEW

File: apps/loop/app.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default () => ({
     name: name,
   },
   environment: {
-    uri: 'https://backend-fusion-data-gateway-test.radix.equinor.com',
```
Comment (Gustav-Eikaas) [inline apps/loop/app.config.js:11]: revert

File: libs/loopapp/src/lib/config/tableConfig.tsx [hunk 2]
```diff
@@ -31,12 +32,43 @@ export const useTableConfig = (contextId: string): GridConfig<Loop, FilterState>
     };
   }, columnDefinitions);
 
+  async function fetchLoopExport(filterstate: FilterState): Promise<any> {
+    const requestOptions = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/problem+json',
```
Comment (Gustav-Eikaas) [inline libs/loopapp/src/lib/config/tableConfig.tsx:39]: revert to application/json

File: libs/loopapp/src/lib/config/tableConfig.tsx [hunk 3]
```diff
@@ -31,12 +32,43 @@ export const useTableConfig = (contextId: string): GridConfig<Loop, FilterState>
...
+    const blob = await res.blob();
+
+    const url = window.URL.createObjectURL(
+      new Blob([blob], {
+        type: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+      })
+    );
+
+    window.open(url, '_self');
+    window.URL.revokeObjectURL(url);
```
Comment (Gustav-Eikaas) [inline libs/loopapp/src/lib/config/tableConfig.tsx:62]: clever code ;)

**Comment by espenkalle** [comment]
deploy test

**Comment by espenkalle** [comment]
deploy test

File: libs/loopapp/src/lib/config/tableConfig.tsx [hunk 4]
```diff
@@ -31,12 +32,43 @@ export const useTableConfig = (contextId: string): GridConfig<Loop, FilterState>
     };
   }, columnDefinitions);
 
+  async function fetchLoopExport(filterstate: FilterState): Promise<any> {
```
Comment (Gustav-Eikaas) [inline libs/loopapp/src/lib/config/tableConfig.tsx:35]: Promise<void>

**Review by Gustav-Eikaas — APPROVED** [review]

**Pull request merged**